nemo/lightning/io/pl.py

```diff
@@ -64,6 +69,22 @@ def construct_extra(cls, trainer: pl.Trainer) -> Dict[str, Any]:
         return extra
 
 
+def ckpt_to_weights_subdir(filepath: Union[str, Path], is_saving: bool = False) -> Path:
```

Comment:
let's drop the default value for `is_saving` and force the user to give it as an input, since this is a critical code part I want to avoid accidental errors from copy/paste.